### ISSUE TITLE
USWDS-Site: Draft content for icon list accessibility checklist page

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds-site#2985]
## Summary
Draft the website content for the icon list accessibility checklist items in the [Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing) (Google Doc :lock:)

## Tasks
- [x] Draft checklist content and send for review with A11Y
- [x] Final edits to checklist content

**Comment by finekatie** [comment]
Icon list is ready for review, @alex-hull !

**Comment by alex-hull** [comment]
@amyleadem @RachelCorsino @cathybaptista @mlloydbixal I have reviewed this, and should be ready for y'all! Marking as done.
